packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts

```diff
@@ -67,12 +67,14 @@ export class ComponentUtils {
       if (!isNullOrUndefined(cmpArray)) {
         cmpArray = cmpArray instanceof Array ? cmpArray : [cmpArray];
         for (const cmp of cmpArray) {
-          const { fullName, manageableState } = cmp;
+          const { fullName, manageableState, namespacePrefix } = cmp;
           if (
             !isNullOrUndefined(fullName) &&
             validManageableStates.has(manageableState)
           ) {
-            components.push(fullName);
+            components.push(
```

Comment:
Is this going to be the label in the tree? If so, is there an alternative to keeping track of namespace and track namespace as part of the tree node?

Reply:
I didn't get the follow up question, but yes, it would be the new label in the tree

Reply:
Do we still need this change?